packages/ds-healthcare-gov/src/components/Header/__snapshots__/Header.test.jsx.snap

```diff
@@ -29,12 +29,12 @@ exports[`Header renders Direct Enrollment banner 1`] = `
           Array [
             Object {
               "href": "https://www.cuidadodesalud.gov/es/",
-              "label": "header.español",
+              "label": "header.español | {\\"ns\\":\\"en\\"}",
```

Comment:
I updated the translation mock so that it prints out the extra options. That's why these snapshots are changing. It's showing that it's looking for this translation key in the `en` namespace